cwms-data-api/src/test/resources/cwms/cda/data/dto/stream.json

```diff
@@ -1,26 +1,30 @@
 {
   "starts-downstream": true,
-  "flows-into-stream": {
+  "flows-into-stream-node": {
     "stream-id": {
-      "office-id": "Office123",
+      "office-id": "SPK",
       "name": "AnotherStream"
     },
     "station": 123.45,
-    "bank": "L"
+    "bank": "L",
+    "station-unit": "ft"
   },
-  "diverts-from-stream": {
+  "diverts-from-stream-node": {
     "stream-id": {
-      "office-id": "Office123",
+      "office-id": "SPK",
       "name": "UpstreamStream"
     },
     "station": 678.9,
-    "bank": "R"
+    "bank": "R",
+    "station-unit": "ft"
   },
   "length": 10.5,
   "slope": 0.01,
+  "length-unit": "mi",
```

Comment:
You have different units for station and length. That is not supported in the DB API where length and station are in the same units for both retrieval and storage:
 * @param p_length               The length of the stream, in station unit
 ...
 * @param p_station_unit         The unit for stream stationing

Reply:
updated json to reflect DB where length and station are always the same unit